woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -316,3 +317,45 @@ def test_age_validate(sample_df, logical_type):
 
     actual = logical_type.validate(series, return_invalid_values=True)
     assert to_pandas(actual).equals(to_pandas(invalid_row))
+
+
+def test_phone_number_validate(sample_df):
+    phone_number = PhoneNumber()
+    dtype = phone_number.primary_dtype
+    series = sample_df["phone_number"].astype(dtype)
+    invalid_row = pd.Series({4: "bad_phone"}, name="phone_number").astype(dtype)
+
+    if _is_spark_series(series):
+        invalid_row = ps.from_pandas(invalid_row)
+
+    assert phone_number.validate(series) is None
+
+    series = series.append(invalid_row).astype(dtype)
+    match = "Series phone_number contains invalid phone number values. "
+    match += "The phone_inference_regex can be changed in the config if needed."
+
+    with pytest.raises(TypeValidationError, match=match):
+        phone_number.validate(series)
+
+    actual = phone_number.validate(series, return_invalid_values=True)
+    expected = pd.Series({4: "bad_phone"}, name="phone_number").astype(dtype)
+    assert to_pandas(actual).equals(expected)
+
+
+def test_phone_number_validate_complex(sample_df_phone_numbers):
+    phone_number = PhoneNumber()
+    dtype = phone_number.primary_dtype
+    series = sample_df_phone_numbers["phone_number"].astype(dtype)
+    # Current inference function does not match lack of area code
+    invalid_row = pd.Series(
+        {17: "252 9384", 18: "+1 194 129 1991", 19: "+001 236 248 8482"},
+        name="phone_number",
+    ).astype(dtype)
```

Comment:
Do we support phone numbers with country codes?

Reply:
For the first pass, we do not. We can create an issue or wait for a Woodwork user to run into an issue.

Reply:
https://www.oreilly.com/library/view/regular-expressions-cookbook/9781449327453/ch04s02.html

Reply:
Yes this supports country codes but the only one we currently support is +1 which is USA/Canada (and several Caribbean nations)

Reply:
Ah okay, that makes sense. Thanks for explaining!